src/components/tabs/TabsBar.js

```diff
@@ -1,7 +1,9 @@
+import touch from '~util/touch'
+
 export default {
   name: 'tabs-bar',
 
-  inject: ['isScrollable'],
+  inject: ['isScrollable', 'isMobile'],
```

Comment:
This will not be reactive since it is an inject. I'm not sure if it matters to support the resizing of the window.